requirements.txt

```diff
@@ -81,13 +77,16 @@ sqlalchemy==1.3.24
     # via
     #   indexd (setup.py)
     #   sqlalchemy-utils
-sqlalchemy-utils==0.41.1
+sqlalchemy-utils==0.41.2
     # via indexd (setup.py)
+sqlparse==0.4.4
```

Comment:
SNYK is not happy with this one

Reply:
https://pypi.org/project/sqlparse/#history

need python3.8 to upgrade this.